.changeset/cyan-hounds-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': minor
+---
+
+Interactive elements will now be correctly focused on Safari.
```

Comment:
Maybe the version can be mentioned here.

Reply:
Oop, your comment came in at the same time as I hit "merge" 😅 We can update the changelog before making the final release.

Reply:
Oop, your comment came in at the same time as I hit "merge" 😅 We can update the changelog before making the final release.

Can you leave a comment on #1853?